### PR TITLE
documentation for job service

### PR DESCRIPTION
**Pull request by luis-soares-ritense**

### CODE REVIEW

**Comment by luis-soares-ritense** [comment]
Waiting for https://github.com/valtimo-platform/valtimo-backend-libraries/pull/649 merge

File: using-valtimo/process/process-beans/job-service.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Job service
+
+This service provides a way to change Timer Events withing the current process
```
Comment (marijnritense) [inline using-valtimo/process/process-beans/job-service.md:3]: I think the name JobService is so generic we should instead say the jobservice is all about manipulating jobs.

File: using-valtimo/process/process-beans/job-service.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### Changing Timer event date  
+
+Job service provides two methods that allow for an update on the timer due date in the current process
```
Comment (marijnritense) [inline using-valtimo/process/process-beans/job-service.md:7]: What are these useful for? I think we should give an example.

File: using-valtimo/process/process-beans/job-service.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+${jobService.updateTimerDueDateByActivityId("2300-01-01T00:00:00Z","timer-id",execution)} 
+${jobService.addOffsetInMillisToTimerDueDateByActivityId(1000,"timer-id",execution)};
+```
+
+"updateTimerDueDateByActivityId" - allows changing the date by passing it as "String", in the ISO 8601 format , as the
```
Comment (marijnritense) [inline using-valtimo/process/process-beans/job-service.md:14]: Not consistent with how we write these things down. Please look at other pages.

**Comment by marijnritense** [comment]
I'm missing a lot of punctuation. Can you go over it one more time and make sure every sentence at least ends with a period?

File: using-valtimo/process/process-beans/job-service.md (hunk 2 above)
Comment (luis-soares-ritense) [reply using-valtimo/process/process-beans/job-service.md:7]: well i understand and agree but then again i really don't know a specific circumstance where this is useful.

**Review by marijnritense — APPROVED** [review]

**Pull request merged**